megalinter/descriptors/bicep.megalinter-descriptor.yml

```diff
@@ -5,18 +5,23 @@ descriptor_flavors:
 linters:
   - cli_help_arg_name: --help
     cli_executable: bicep
-    cli_lint_mode: file
+    cli_lint_errors_count: "\\.bicep\\(\\d+,\\d+\\) : Error "
```

Comment:
the properties are cli_lint_errors_count and cli_lint_errors_regex :)
https://github.com/oxsecurity/megalinter/blob/43e281083aea9a875f36f630d924512ea6954ac5/megalinter/descriptors/javascript.megalinter-descriptor.yml#L49

Reply:
cli_lint_error_count can be
- regex_sum ( sum all the numbers found by regex in logs)
- regex_number (find a single number with regex in logs)
- regex_count (count the number of times the regex is matched in the log)